apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.BenchmarkingTool/Program.cs

```diff
@@ -0,0 +1,244 @@
+
+                for (int i = 1; i <= configuration.NumOfSamples; ++i)
+                {
+                    Console.WriteLine($"Running W1R3 iteration number {i}..");
+                    RunW1R3Benchmark(configuration, client, logger);
+                }
+
+                s_stopwatch.Stop();
+            }
+            catch (ArgumentException e)
+            {
+                var originalColour = Console.ForegroundColor;
+                Console.ForegroundColor = ConsoleColor.Red;
+                Console.Write(e.Message);
+                Console.ForegroundColor = originalColour;
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+            }
+        }
+
+        private static void RunW1R3Benchmark(Configuration configuration, StorageClient client, PerformanceLogger logger)
+        {
+            try
+            {
+                CreateBucket(configuration.Bucket, client);
+
+                Write(configuration, client, logger, "Write");
+                Read(configuration, client, logger, "Read[1]");
+                Read(configuration, client, logger, "Read[2]");
+                Read(configuration, client, logger, "Read[3]");
+            }
+            finally
+            {
+                DeleteBucket(client, configuration.Bucket);
+            }
+        }
+
+        private static Bucket CreateBucket(string bucketName, StorageClient client)
+        {
+            var bucket = client.CreateBucket(s_projectId, new Bucket { Name = bucketName });
+            SleepAfterBucketCreateDelete();
+            return bucket;
+        }
+
+        private static void DeleteBucket(StorageClient client, string bucketName)
+        {
+            try
+            {
+                client.DeleteBucket(bucketName, new DeleteBucketOptions { UserProject = null, DeleteObjects = true });
+            }
+            catch (GoogleApiException)
+            {
+                //Some tests fail to delete buckets due to object retention locks etc.
+                //They can be cleaned up later.
+            }
+            SleepAfterBucketCreateDelete();
+        }
+
+        private static void CreateRandomFile(Configuration configuration)
+        {
+            var random = new Random();
+            long bytesLeft = configuration.DataSizeInBytes;
+            byte[] buffer = new byte[1024 * 1024];
+            using var output = File.Create(configuration.LocalFile);
+            while (bytesLeft > 0)
+            {
+                random.NextBytes(buffer);
+                int count = (int) Math.Min(bytesLeft, buffer.Length);
+                output.Write(buffer, 0, count);
+                bytesLeft -= count;
+            }
+        }
+
+        private static void Write(Configuration configuration, StorageClient client, PerformanceLogger logger, string opName)
+        {
+            string elapsedTimeUs = "-";
+            bool isSucccess = false;
```

Comment:
Suggest renaming to just "success" (note two c's rather than three, as well as removing the "is").